docs/products.md

```diff
@@ -2,8 +2,55 @@
 
 ## RTC
 
+SAR datasets inherently contain geometric and radiometric distortions due to terrain
+being imaged by a side-looking instrument. Radiometric terrain correction (RTC) removes
+these distortions and creates analysis-ready data suitable for use in GIS applications.
+RTC processing is a required first step for many amplitude-based SAR applications.
+
+Sentinel-1 RTC products are generated by ASF's HyP3 processing platform leveraging
+GAMMA Software.  Products are distributed as UTM-projected GeoTIFFs with a pixel
+spacing of 30 meters. To learn more, visit the
+[ASF Sentinel-1 RTC Product Guide](guides/rtc_user.md).
+
+A Digital Elevation Model (DEM) is required for processing InSAR, and ASF uses the
+best publicly-available DEM with full coverage of the processing area,
+see [Digital Elevation Model](dems.md) for more information.
```

Comment:
I slightly prefer `to learn more, visit...` over `for more information, see...`.  It feels a little more engaging and conversational.